content/en/docs/contribute/generate-ref-docs/contribute-upstream.md

```diff
@@ -91,7 +91,7 @@ will be different in your situation.
 
 Here's an example of editing a comment in the Kubernetes source code.
 
-In your local kubernetes/kubernetes repository, check out the master branch,
+In your local kubernetes/kubernetes repository, check out the primary branch,
```

Comment:
👋 @sftim .
Could this sentence continue to refer to the `master` branch but indicate that these instructions (prefer or give preference to) will refer to this branch as the primary branch?

Reply:
Yep, that could work.